src/commands/upload_package_using_cli.yml

```
@@ -0,0 +1,22 @@
+description:
+  Generate temporary Cloudsmith credentials using OIDC and upload a java maven package to a Cloudsmith repository using the
+  Cloudsmith Native CLI.
+parameters:
+  repository:
+    description: The identity/slug of the Cloudsmith repository
+    type: string
+  dist_dir:
+    description: The distribution directory where package source distribution and wheel files are located
```

```suggestion
    description: The distribution directory where package source distribution files are located
```
I don't think maven uses wheel files.